examples/hello_world/disagg_skeleton/components/prefill_worker.py

```
@@ -60,46 +60,52 @@ async def async_init(self):
         def prefill_queue_handler_cb(fut):
             try:
                 fut.result()
-                print("prefill queue handler exited successfully")
+                logger.info("prefill queue handler exited successfully")
             except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)